.travis.yml

```diff
@@ -7,6 +7,10 @@ scala:
   - 2.10.5
   - 2.11.7
 
+cache:
+  directories:
+  - $HOME/.ivy2
```

Comment:
Maybe it would make sense to cache `sbt` as well (`$HOME/.sbt`). I found this:
```
...
Next, we can put cache section as follows:

# These directories are cached to S3 at the end of the build
cache:
  directories:
    - $HOME/.ivy2/cache
    - $HOME/.sbt/boot/
Finally, the following a few lines of cleanup script are added:

before_cache:
  # Tricks to avoid unnecessary cache updates
  - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
  - find $HOME/.sbt -name "*.lock" -delete

```
in [SBT docs](http://www.scala-sbt.org/0.13/docs/Travis-CI-with-sbt.html)